client/src/App.jsx

```diff
@@ -15,6 +15,8 @@ import LoginPage from "./pages/login";
 import CreatePage from "./pages/create-user";
```

Comment:
Hi Daniel, thank you so much for the catches and sharing the error prompt. I have pushed changes to fix the issues listed above. Modify status and closing should behave as normally now.

However, for the point "The table currently shows "Unknown" as the owner of any client who User 2 owns". The reason is because, we actually deleted the user with id 2 from our user table from the admin dashboard a while back so user with id 2 no longer exists. That's why if a user with a certain id can't be found, we just display it   as unknown for now. Let me know if this is fine.

Reply:
Hey Kevin, that sound totally reasonable, thanks for the explanation 👍